nrel/hive/model/roadnetwork/osm/osm_builders.py

```diff
@@ -13,6 +13,7 @@ def osm_graph_from_polygon(polygon):
         ) from e
 
     ox.settings.all_oneway = True
+    ox.settings.use_cache = False
```

Comment:
why not use cache?

Reply:
I wonder if we could set the cache folder instead to cache the road network results in the scenario folder using the `cache_folder` setting from [here](https://osmnx.readthedocs.io/en/stable/osmnx.html?highlight=config#module-osmnx.settings)?

Reply:
Talked with Nick and we got this updated, unsure if I love the implementation though. Open to suggestions